inc/template.php

```diff
@@ -53,6 +55,7 @@ function get_authors( WP_Post $post ) : array {
 
 	/** @var WP_User[] */
 	$users = get_users( [
+		'blog_id' => 0,
```

Comment:
Does this mean any site or just authors with no site? Have you tested with a combination of users with a role on the current site and without?

Reply:
Correct, tested it with multiple users assigned to a site and not assigned. By default `blog_id` uses the current site. While there is no official docs on using the `0` for the `blog_id` there is [mention of it in the comments](https://developer.wordpress.org/reference/classes/wp_user_query/prepare_query/#comment-2248).

Reply:
Great find! It'd be good to leave a comment in the code to that effect, or referencing that comment for future maintainers to understand & learn from.

Reply:
For comments like this, is a single inline comment suitable, or should it be done in a multiline comment before `$users = get_users( [`?

Reply:
Single line is fine, but it doesn't really matter too much. Don't overthink it :)